db/models.py

```diff
@@ -1 +1,28 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
Consider using a more appropriate field type for the `bonus` attribute if it is meant to store numerical values or specific types of bonuses. `CharField` may not be the best choice if the data has a specific format or type.